temba/settings.py.prod

```diff
@@ -77,6 +77,7 @@ REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.messages"] = env("V2_MESSAGES", def
 REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.broadcasts"] = env("V2_BROADCASTS", default="36000/hour")
 REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.runs"] = env("V2_RUNS", default="3000/hour")
 REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]["v2.api"] = env("V2_API", default="3000/hour")
+REST_FRAMEWORK["PAGE_SIZE"] = 100
```

Comment:
pls change the integer defined to an environment variable, with a value default with 250, and on configmap add the number 100

Reply:
the default value shouldn't be 100 for the environment variable @matmsa27 ? since this change is to reduce this value

Reply:
Since we are testing this change performance impact, i think keeping 250 as default is the better option.

Reply:
let's keep 250 with value default, we can treat this on the application environment.